src/rest/main/main.go

```diff
@@ -95,17 +104,17 @@ func main() {
 		}
 	}()
 
-	if caFile == "" && server.ClientAuth.Enabled("cert") {
+	if caFile == "" && clientAuth.Enabled("cert") {
 		glog.Fatal("Must specify -cacert with -client_auth cert")
 	}
 
 	swagger.Load()
 
 	server.GenerateJwtSecretKey()
-	server.JwtRefreshInt = time.Duration(jwtRefInt * uint64(time.Second))
-	server.JwtValidInt = time.Duration(jwtValInt * uint64(time.Second))
+	server.JwtRefreshInt = time.Duration(30 * time.Second)
+	server.JwtValidInt = time.Duration(3600 * time.Second)
 
-	router := server.NewRouter()
+	router := server.NewRouter(clientAuth)
```

Comment:
Need to account for the upgrade scenario where "user" maybe specified instead of "password"

Reply:
Added, see my latest commit https://github.com/project-arlo/sonic-mgmt-framework/pull/504/commits/3dc81d0b40fb31cb3b53db56f725a084cfff51e9